demo-app/src/open-truss/components/NextLink.tsx

```diff
@@ -1,11 +1,11 @@
 import {
-  withChildren,
+  withChildrenV1,
```

Comment:
fwiw I don't love versioning `withChildren`. It's such a fundamental piece of React I can't imagine it changing

Reply:
Done in [c3f6e98](https://github.com/open-truss/open-truss/pull/94/commits/c3f6e98ed59d62172e0a2b7144c9eea5e2799ffd)